hack/container_build_scripts/utils.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+function replace_dnf_repositories_ref () {
```

Comment:
Maybe put a comment above this function to explain why we are doing this?
I guess that's reasonable in this case, especially as we are doing it as a NO-ISSUE?